.env-dist

```diff
@@ -41,19 +41,20 @@ BASKET_NEWSLETTER=mozilla-and-you
 # leave FXA_ENABLED empty to disable FXA
 FXA_ENABLED=
 FXA_SETTINGS_URL="https://stable.dev.lcip.org/settings"
-OAUTH_CLIENT_ID=cb1bef9d06bb9bc9
-OAUTH_CLIENT_SECRET=f5fb99de6e0af18ab17e013ac1d439903179a97a1c510fc10bc3bd50bbce089b
+OAUTH_CLIENT_ID=edd29a80019d61a1
+OAUTH_CLIENT_SECRET=a80feaad77c847275d39ac989ee12a873ef6b54cbc184128f86f2afecdf003b5
```

Comment:
I assume these are OK to change? (I know it's just a dist file, but wanted to confirm these aren't critical creds that need to be regenerated now).

Reply:
Yup, these are the values I have on my local file and it's working.